app/models/user.rb

```diff
@@ -290,7 +290,7 @@ def power_of_attorney
   end
 
   def flipper_id
-    "User;#{account_uuid}"
+    email || account_uuid
```

Comment:
Shouldn't it be `"User;#{email || account_uuid}"` is there a reason for why it was formatted like that previously?